qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManagePrivilegesPage.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+
+public class ManagePrivilegesPage extends Page{
+
+	protected static final String PAGE_URL = "adminui/metadata/privileges/managePrivileges.page";
+	private static final By ADD_NEW_PRIVILEGE_BUTTON = By.cssSelector("#content > input");
+	private static final By EDIT_PRIVILEGE_ICON = By.cssSelector("i.icon-pencil.edit-action");
+	private static final By DELETE_ICON = By.cssSelector("i.icon-trash.delete-action.right");
+	private static final By SEARCH_FIELD = By.cssSelector("#list-privileges_filter > label > input[type=text]");
+	private static final By CONFIRM_BUTTON = By.cssSelector("#purgePrivilegeForm > div > button.confirm.right");
+	
+	public ManagePrivilegesPage(Page configureMetadataPage) {
+		super(configureMetadataPage);
```

Comment:
I know configureMetadataPage is just a variable name however, what is the super page class to ManagePriviliges?